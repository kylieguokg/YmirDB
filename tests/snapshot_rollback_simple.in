rollback 0
set a 1 2 3
list entries
snapshot
set b 29 38 10
list entries
snapshot
set c -1 -2 -3
list entries
snapshot
rollback 1
list entries
set new 2 3 0
list entries
snapshot
checkout 2
checkout 3
checkout 4
list entries
bye
