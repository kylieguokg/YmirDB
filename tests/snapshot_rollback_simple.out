> no such snapshot

> ok

> a [1 2 3]

> saved as snapshot 1

> ok

> b [29 38 10]
a [1 2 3]

> saved as snapshot 2

> ok

> c [-1 -2 -3]
b [29 38 10]
a [1 2 3]

> saved as snapshot 3

> ok

> a [1 2 3]

> ok

> new [2 3 0]
a [1 2 3]

> saved as snapshot 4

> no such snapshot

> no such snapshot

> ok

> new [2 3 0]
a [1 2 3]

> bye
